src/content/services/contentScript.ts

```diff
@@ -58,4 +59,14 @@ export class ContentScript {
       }
     )
   }
+
+  // TODO: When Firefox supports `world: MAIN`, inject this directly via
```

Comment:
It looks like we might get this in the next release - Firefox 128? :eyes: 
https://bugzilla.mozilla.org/show_bug.cgi?id=1736575

What's going to be the issue if we add it with `world: 'MAIN'` right now?

Reply:
Yes, I'm following that issue.

If we add `world: MAIN` now: install time warnings and `window.Monetization` not available.

Reply:
(But we plan to support some older versions I think, so, might have to wait a year). I'll add the bugzilla link in a comment.